File: bouncing-balls/bouncing-balls.js
function bouncingBall(h,  bounce,  window) {
if( h < 0 || bounce>=1 || bounce<=0 || window>h || window<=0){return -1}
var y = 0
    while(h > window){
        h = h * bounce
        y += 1}
return 2*(y-1)+1
}

File: bouncing-balls/bouncing-balls.py
def bouncingBall(h, bounce, window):
    if h < 0 or bounce>=1 or bounce<=0 or window>h or window<=0:
        return -1
    high = h 
    y = 0
    while high > window:
        high = high * bounce
        y += 1
        
    return 2*(y-1)+1